airzone/localapi.py

```diff
@@ -93,72 +95,70 @@ def set_zone_parameter_value(self, zone_id, parameter, value):
             self._data[parameter] = value
             response = requests.put(url=self._API_ENDPOINT,
                                     json=self._data)
-            self.handle_response(response)            
+            self.handle_response(response)
         except requests.exceptions.RequestException as e:
             raise SystemExit(e)
-    
+
     def _get_zone_property(self, zone_id, prop):
         z_id = zone_id
         if z_id == 0:
-            z_id = next(iter(self._zones))            
+            z_id = next(iter(self._zones))
         if prop in self._zones[z_id].zone_state:
-            return self._zones[z_id].zone_state[prop]        
+            return self._zones[z_id].zone_state[prop]
         return None
 
-
-
     @property
     def speed(self):
         speed = self._get_zone_property(0, 'speed')
-        if speed != None:
+        if speed is not None:
             return Speed(speed)
         return Speed.AUTO
-    
+
     @speed.setter
     def speed(self, speed):
         self.set_zone_parameter_value(0, 'speed', speed)
-    
+
     @property
-    def operation_mode(self):       
+    def operation_mode(self):
         return OperationMode(self._get_zone_property(0, 'mode'))
 
     @operation_mode.setter
     def operation_mode(self, mode):
         self.set_zone_parameter_value(0, 'mode', mode)
-    
-    @property    
+
+    @property
     def units(self):
         return TempUnits(self._get_zone_property(0, 'units'))
 
     def unique_id(self):
         return f'Local_Api{self._machine_id}_{str(self._machine_ip)}'
-  
+
     def __str__(self):
         zs = "\n".join([str(z) for z in self.get_zones()])
         return "Machine with id: " + str(self._machine_id) + \
-                "Mode: " + str(self.operation_mode) + \
-                "\nZones\n" + zs
+               "Mode: " + str(self.operation_mode) + \
+               "\nZones\n" + zs
 
 
 class Zone:
-    def __init__(self, machine, zone_id, data = {}):
+    def __init__(self, machine, zone_id, data={}):
         self._machine = machine
-        self._zone_id = zone_id        
-        self.zone_state = data
-              
+        self._zone_id = zone_id
+        self._zone_state = data
```

Comment:
The property must be used instead of the inner variable.
Only use the inner if there is a proper reason to do.
So this has to be ```self.zone_state = data```